feeder/parsers/ENTSOE.py

```diff
@@ -72,11 +72,13 @@ def query_consumption(domain, session):
         'outBiddingZone_Domain': domain,
         'periodStart': now.replace(hours=-24).format('YYYYMMDDHH00'),
         'periodEnd': now.replace(hours=+24).format('YYYYMMDDHH00'),
-        'securityToken': os.environ['ENTSOE_TOKEN']
+        'securityToken': os.environ.get('ENTSOE_TOKEN')
```

Comment:
This is equivalent to the previous formulation. Let's keep the shorter version instead!